packages/core/src/provider/base.ts

```diff
@@ -267,6 +267,10 @@ export class SmartAccountProvider<
     struct: UserOperationStruct
   ): Promise<UserOperationStruct> => {
     struct.paymasterAndData = "0x";
+    // by default add a 25k overhead to verification gas limit
+    struct.verificationGasLimit =
```

Comment:
This is no longer needed, rundler does this automatically

Reply:
ok cool will drop this and then it should be sufficient to just support overriding gas on paymaster middleware